source/fundamentals/builders/aggregates.txt

```diff
@@ -19,7 +19,10 @@ In this guide, you can learn how to use the `Aggregates <{+api+}/apidocs/mongodb
 class which provides static factory methods that build :manual:`aggregation pipeline
 stages </meta/aggregation-quick-reference/#stages>` in the MongoDB Java driver.
 
-For a more thorough introduction to Aggregation, see our :ref:`Aggregation guide <java-aggregation>`.
+These builders serve as a wrapper for aggregation expressions and provide some
+aggregation functionality. For a more thorough introduction to Aggregation, see
+our :ref:`Aggregation guide <java-aggregation>`. For a more complete syntax
```

Comment:
Issue:
To me, "for a more complete...enables greater aggregation functionality" implies that the Aggregation Expressions subsumes the content on this page, which I don't think is the case.

Suggestion:
Omit this paragraph.

Reply:
I think it is important for people to have a link to the greater topic on aggregation, and also important for folks to know that this is one solution, while there is another. I did some rewording, but I think we need something here.